BitwardenShared/Core/Platform/Services/AuthenticatorSyncService.swift

```diff
@@ -55,6 +55,10 @@ actor DefaultAuthenticatorSyncService: NSObject, AuthenticatorSyncService {
     /// a user opts-in to Authenticator sync.
     private var syncSettingSubscriberTask: Task<Void, Error>?
 
+    /// a Task that subscribes to the vault lock publisher for accounts. This allows us to take action once
+    /// a user unlocks their vault..
+    private var vaultUnlockSubscriberTask: Task<Void, Error>?
+
```

Comment:
🤔 Are these tasks going to be used for something else in a future PR? Because on the current code we could avoid having these variables.

Reply:
I don't think I have a clear vision for them in a future PR. You're suggesting just eliminating the `private var` reference to them and just declare them inline as Tasks without a reference? I think that should be fine. 🤔  I'll take a look at that.

Reply:
Yep, indeed. If we're not going to handle cancellation or something else with this task reference then we could remove these private var references.

Reply:
Update to this in the latest commit. 👍